e2e/test/DeviceStreamingE2ETests.cs

```diff
@@ -45,6 +45,19 @@ public async Task DeviceStreaming_RequestAccepted_Sas_Amqp()
             await TestDeviceStreamingAsync(TestDeviceType.Sasl, transportSettings, true).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        public async Task DeviceStreaming_RequestAccepted_Sas_Amqp_WithProxy()
+        {
+            Client.AmqpTransportSettings amqpTransportSettings = new Client.AmqpTransportSettings(Client.TransportType.Amqp_Tcp_Only)
```

Comment:
The test doesn't match the code change above: this is setting AMQP proxy and above is HTTP.